src/main/java/beam/physsim/jdeqsim/AgentSimToPhysSimPlanConverter.java

```diff
@@ -171,6 +173,14 @@ private void setupActorsAndRunPhysSim(int iterationNumber) {
             }
         }
 
+        // Black magic happens here
```

Comment:
`router.tell(new BeamRouter.UpdateTravelTime(travelTimeCalculator.getLinkTravelTimes()), ActorRef.noSender());` is still happening below - do we want to have both ways - maybe configurable - or local gets the uncompressed version? Also, shouldn't this be moved down to that same end of method location?

Reply:
It's a good idea to keep both ways (optimal for local router), I'll make changes. Regarding to moving `router.tell(new BeamRouter.UpdateTravelTime_v2(map), ActorRef.noSender());`, I left it there because `linkStatsGraph.notifyIterationEnds` will start changing `travelTimeCalculator`, if there were no links